packages/x-tree-view/src/internals/plugins/useTreeViewSelection/useTreeViewSelection.types.ts

```diff
@@ -94,6 +99,23 @@ export interface UseTreeViewSelectionParameters<Multiple extends boolean | undef
    * @default false
    */
   checkboxSelection?: boolean;
+  /**
+   * When `rowSelectionPropagation.descendants` is set to `true`.
+   *
+   * - Selecting a parent selects all its filtered descendants automatically.
+   * - Deselecting a parent row deselects all its filtered descendants automatically.
+   *
+   * When `rowSelectionPropagation.parents` is set to `true`.
+   *
+   * - Selecting all the filtered descendants of a parent selects the parent automatically.
+   * - Deselecting a descendant of a selected parent deselects the parent automatically.
```

Comment:
```suggestion
   * - Selecting all the descendants of a parent selects the parent automatically.
   * - Deselecting a descendant of a selected parent deselects the parent automatically.
```

Reply:
You are right, it was taking on the grid, but the tree view do not use inline background on other demos